packages/cli/package.json

```diff
@@ -70,16 +70,21 @@
     "@types/convict": "^4.2.1",
     "@types/cookie-parser": "^1.4.2",
     "@types/express": "^4.17.6",
+    "@types/ioredis": "^4.0.0",
```

Comment:
we upgrade ioredis last week. the updated package comes with it's own typings, and we should remove this stub package.